XiEditor/LineCache.swift

```diff
@@ -286,19 +286,21 @@ class LineCacheLocked<T> {
             .map( { $0.element })
         if !missingLines.isEmpty {
             // TODO: should we send request to core?
+#if DEBUG
             print("waiting for lines: (\(missingLines.first!), \(missingLines.last!))")
+#endif
             //TODO: this timing + printing code can come out
             // when we're comfortable with the performance and
             // the timeout duration
-            let blockTime = mach_absolute_time()
+            let blockTime = DispatchTime.now()
```

Comment:
Should I be preferring this to `mach_absolute_time()` in general?

Reply:
Yes.